sql/catalyst/pom.xml

```diff
@@ -60,6 +60,11 @@
       <artifactId>scalacheck_${scala.binary.version}</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
```

Comment:
I think that our motivation for keeping the versions in the root was to avoid having different subprojects depending on different versions of the same library.  That's probably not an issue for janino, so I guess that moving the version would only be for consistency.